demo/index.html

```diff
@@ -1,54 +1,74 @@
 <!doctype html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
-  <title>vaadin-combo-box Examples</title>
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1"
+      />
+    <title>vaadin-combo-box Examples</title>
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 
-  <link rel="import" href="../../polymer/polymer.html">
-  <link rel="import" href="../../vaadin-demo-helpers/vaadin-component-demo.html">
-  <link rel="import" href="../../vaadin-demo-helpers/vaadin-demo-snippet.html">
-  <script src="../../vaadin-demo-helpers/vaadin-demo-ready-event-emitter.js"></script>
+    <link rel="import" href="../../polymer/polymer.html">
+    <link rel="import" href="../../vaadin-demo-helpers/vaadin-component-demo.html">
+    <link rel="import" href="../../vaadin-demo-helpers/vaadin-demo-snippet.html">
+    <script src="../../vaadin-demo-helpers/vaadin-demo-ready-event-emitter.js"></script>
+    <script src="combo-box-demo.js"></script>
 
-  <script src="combo-box-demo.js"></script>
+    <link rel="import" href="../vaadin-combo-box-light.html">
+    <link rel="import" href="../../paper-icon-button/paper-icon-button.html">
+    <link rel="import" href="../../iron-ajax/iron-ajax.html">
+    <link rel="import" href="../../iron-icon/iron-icon.html">
+    <link rel="import" href="../../iron-icons/iron-icons.html">
+    <link rel="import" href="../../iron-input/iron-input.html">
+    <link rel="import" href="../../paper-input/paper-input.html">
+    <link rel="import" href="../../vaadin-button/vaadin-button.html">
+    <link rel="import" href="../vaadin-combo-box.html">
+    <link rel="import" href="../../iron-form/iron-form.html">
```

Comment:
Since iron form example was removed, the import and the dependency should be removed also.

Reply:
Looks like this applies to all iron- and paper- imports, vaadin-button and even vaadin-combo-box-light

Reply:
I removed <link rel="import" href="../../iron-form/iron-form.html">

Reply:
Nope, I still see them in index.html and bower.json (devDependencies, some are still needed, at least iron-component-page and maybe iron-test-helpers)